Add blog about changing logging level of a WildFly application on the Cloud

jmesnil opened the pull request:
Signed-off-by: Jeff Mesnil <jmesnil@redhat.com>

jmesnil commented:
@jamezp could you please review if the logging commands are correct? thanks!

I planned the post for today but there is no rush. If you don't have time to review it today, I'll change the date to post it later next week?

yersan reviewed (commented):
One suggestion, maybe you could mention that the approach is only valid if the logging subsystem is present. I mean, in the case of a trimmed server, the logging subsystem could be not installed.

jamezp reviewed (commented):
This looks good to me @jmesnil on the technical level. @yersan seems to have spotted a few things :)

I don't know if it's worth mentioning, but should there be an example of debugging a specific logger?

```
/subsystem=logging/logger=org.example.web.app:add(level=DEBUG)
```

jmesnil commented on _posts/2021-06-11-change-log-level-wildfly-cloud.adoc:
```diff
@@ -0,0 +1,123 @@
+author: jmesnil
+---
+
+When your WildFly application is running on the Cloud (on Kubernetes, OpenShift or any other container plaform) and starts to behave unexpectedly, it is useful to be able to change the logging of the server to display debug logs to get a better insight.
```
fixed, thanks

jmesnil commented on _posts/2021-06-11-change-log-level-wildfly-cloud.adoc:
```diff
@@ -0,0 +1,123 @@
+
+## JBOSS_HOME environment variable
+
+These commands works fine whent the application image has been built with https://github.com/wildfly/wildfly-s2i[WildFly S2I] as the `JBOSS_HOME` environment variable is set and we can invoke `$JBOSS_HOME/bin/jboss-cli.sh` without any issue.
```
fixed, thanks

yersan replied:
Somehow, it has not been changed yet, I still see "whent"

jmesnil replied:
sorry, I forgot to save :)

jmesnil commented:
@jamezp good suggestion. I added a note that this script is ham fisted and would likely have to be updated to change log level to specific loggers related to the user application.

I've also updated its date to tomorrow (2021-06-15) for publication.

yersan commented on _posts/2021-06-15-change-log-level-wildfly-cloud.adoc:
```diff
@@ -0,0 +1,131 @@
+
+## JBOSS_HOME environment variable
+
+These commands works fine when the application image has been built with https://github.com/wildfly/wildfly-s2i[WildFly S2I] as the `JBOSS_HOME` environment variable is set and we can invoke `$JBOSS_HOME/bin/jboss-cli.sh` without any issue.
```
s/These commands works/These commands work

jmesnil replied:
thanks!

yersan reviewed (approved):
Ups, just saw an additional minor more, with that one fixed, the content looks fine to me.
